### ISSUE TITLE
Swedish translation not jet fully translated

### ISSUE THREAD

**Issue by XCOM-HUB** [vcmi-mods/swedish-translation#4]
Hi there @Fabrice-TIERCELIN! I use your translation mod for [VCMI](https://heroes3wog.net/vcmi/) 1.5.7.

I wanted to help out. I have already started to help out @CalZing with his translation project here on GitHub (HeroesIIISwedish). Can I start helping you and CalZing to make this translation as good as possible?

**Comment by heroesiiifan** [comment]
@XCOM-HUB 
Of course you can help out to improve everything. Feel free to make a Pull Request. :)

But keep in mind: We've converted the original TXT files to JSON format (native VCMI translation format). So it's maybe a little bit double work to keep them in sync.

I think there was already some adjustments here: https://github.com/vcmi-mods/swedish-translation/pull/3

It would be also great if someone translates the launcher into swedish. Because it's the first thing you see from VCMI, when starting.

@CalZing
A big thank for the project. :)

**Comment by heroesiiifan** [comment]
You can see a progress here: https://github.com/vcmi/vcmi-translation-status

I think it's not 100% for swedish correct because there are still english strings in json file.<

Feel free to ask if you need support.

**Comment by XCOM-HUB** [comment]
I'm new to making Pull Requests, but I'm willing to learn new things. Will VC Code be able to read JSON-files?

**Comment by heroesiiifan** [comment]
Yes of course!

You can basically also use the VS code integrated in GitHub.

Just open https://github.com/vcmi-mods/swedish-translation/blob/vcmi-1.4/swedish-translation/content/config/vcmi-swedish/game.json
and use click on the arrow right of the edit button, then github.dev

or just use the link: https://github.dev/vcmi-mods/swedish-translation/blob/vcmi-1.4/swedish-translation/content/config/vcmi-swedish/game.json

After doing your changes you should do a commit (on 3. icon on left side). Then go back to github. It's asking you to make a Pull request.

**Comment by Fabrice-TIERCELIN** [comment]
Welcome @XCOM-HUB,

Here is a very detailled guide for submitting _Pull Request_ to [misiokles](https://github.com/misiokles):

1. Create the files you want to add/change on the project of a town mod on your computer
1. Go on the [GitHub page of the mod](https://github.com/vcmi-mods/swedish-translation)
2. Click on the [_Code_ tab](https://github.com/vcmi-mods/swedish-translation)
2. Click on the [_Fork_ grey button](https://github.com/vcmi-mods/swedish-translation/fork)
2. Click on the _Create fork_ green button
1. Go to [your repositories page](https://github.com/XCOM-HUB?tab=repositories)
1. Click on your project
3. Click on the _Code_ tab
3. Click on the folders where you want to add/change a file
4. In the right folder, click on the _Add files_ grey button
5. Drag and drop your new/modified files from your computer to the web page
6. Add a commit message
2. Click on the _Commit changes_ green button
2. Click on the _Compare changes_ green button in the brand new orange banner
3. Mention [misiokles](https://github.com/misiokles) in the description
2. Click on the _Create Pull Request_ green button

Once the Pull Request is merged by [misiokles](https://github.com/misiokles), you can remove your fork:

1. Go to [your repositories page](https://github.com/XCOM-HUB?tab=repositories)
1. Click on your project
1. Click on the _Settings_ tab
2. Click on _Delete this repository_
3. Confirm to delete

Now, you are ready to do another Pull Request. Note that this method only allows you to submit files on the same folder of the project.

**Comment by XCOM-HUB** [comment]
Is my recent Pull Request good to go?

**Comment by Fabrice-TIERCELIN** [comment]
I think you are talking about this PR: https://github.com/vcmi-mods/swedish-translation/pull/7

To disambiguate, I advise you to paste the URL of the PR in your comment.

You have clicked on _Close_, which means that any other person should ignore it. If you want your PR to be reconsider, click on _Reopen_. It works but if you want your PR not to be merged for now, you can click on the _draft_ link on the right. Then, when your PR is ready, you click on _Ready for review_.
